### PR TITLE
Lab 4 - numeronyms

### PR DESCRIPTION
**Pull request by loki5265**
Fixes #67

#### Changes proposed in this PR:
* Implement numeronyms method
* Add test case
* Fix code style

### CODE REVIEW

File: 04_numeronym/lokili/main_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+)
+
+var testSet = []struct {
+	input  []string
+	output []string
+}{
+	{[]string{"世界Thrives！"}, []string{"世8！"}},
+	{[]string{""}, []string{""}},
+	{[]string{"&《 44za  "}, []string{"&7 "}},
+	{[]string{"？will他 reach the airport at 10 am ?"}, []string{"？33?"}},
```
Comment (anzboi) [inline 04_numeronym/lokili/main_test.go:17]: You should also test your `numeronyms` function against input arrays of varying length, eg
`[]String{"", "abcd"}`
`[]String{}`

File: 04_numeronym/lokili/main.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(vals ...string) []string {
```
Comment (anzboi) [inline 04_numeronym/lokili/main.go:11]: Perfect 👍

**Review by anzboi — COMMENTED** [review]
Just add a couple of tests and it looks good.

**Comment by juliaogris** [comment]
Sorry, I will close this PR now in preparation for the next training in a few weeks in Melbourne - feel free to re-open it if you want to continue work on it.

**Pull request closed without merging**